models/Freelance.js

```diff
@@ -19,15 +19,21 @@ const ObjectID = mongoose.Schema.Types.ObjectId;
 require ('./Review');
 
 const Freelance = exports.Freelance = new mongoose.Schema({
-		firstName		: { type: String, required: true },
-		familyName		: { type: String, required: true },
-		address			: { type: String },
-		email			: { type: String, required: true },
-		phone			: { type: String },
-		price			: { type: Object, required: true },
-		avgScore 		: { type: Number },
-		reviews			: [{ type: ObjectID, ref: "Review", default: [] }],
-		tags			: [{ type: String, default: [] }],
+		firstName			: { type: String, required: true },
+		familyName		: { type: String },
+		title					: { type: String },
```

Comment:
Can we please make `title` a required property? As said, it's as important as the `firstName`.
A practical reason is: a person that wants to make two profiles for two different jobs, maybe both in the same category.